p2p/event_handler.go

```diff
@@ -40,6 +41,30 @@ func (m *p2pModule) HandleEvent(event *anypb.Any) error {
 			}
 		}
 
+	case messaging.StateMachineTransitionEventType:
+		stateMachineTransitionEvent, ok := evt.(*messaging.StateMachineTransitionEvent)
```

Comment:
https://github.com/pokt-network/pocket/pull/491/files#r1108017327
> NIT: Create small local helper functions for each event type. Will make it easier as we add more events

I see your point but I'd rather wait and see when that happens. In this particular event, it's likely that we'll have to create logical branches depending on the properties of the events, if it starts becoming hairy, I'd refactor. 

For now I think it's simple enough and gives an idea of what it can do.

WDYT?

Reply:
Sgtm. My personal preference would be the other way (easier for me to read) but really don't feel strongly on it this early on. Like you said, prob premature